src/include/parser/expression/constant_value_expression.h

```diff
@@ -35,13 +35,20 @@ class ConstantValueExpression : public AbstractExpression {
     return value_ == const_expr.GetValue();
   }
 
+  void DeriveExpressionName() override {
+    if (!this->GetAlias().empty()) return;
```

Comment:
Just wondering, don't you have to call `this->SetExpressionName(this->GetAlias())`?